### PR TITLE
Add f2fs

### PR DESCRIPTION
**Pull request by SomethingGeneric**
Re-opening from @axtloss but targeting development branch

### CODE REVIEW

**Review by mchalunderscore — APPROVED** [review]

**Comment by mchalunderscore** [comment]
Git stuff, want me to sort it out?

File: chrooted-iso.sh [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
+Exec=sh -c \"gsettings set org.gnome.desktop.screensaver lock-enabled false\"" > /home/crystal/.config/autostart/no-lock-screen.desktop
+
+# Jade-GUI Autostart
+cp \
+  /var/lib/flatpak/exports/share/applications/al.getcryst.jadegui.desktop \
+  /home/crystal/.config/autostart
+
+# Permissions for Crystal user
 chown -R crystal:crystal /home/crystal/
+chmod +x /home/crystal/.config/autostart/*.desktop
```
Comment (axtloss) [inline chrooted-iso.sh:36]: .desktop files usually don't need the executable bit

Comment (mchalunderscore) [reply chrooted-iso.sh:36]: They actually require it for autostart

Comment (axtloss) [reply chrooted-iso.sh:36]: oh interesting

**Review by axtloss — APPROVED** [review]

**Comment by SomethingGeneric** [comment]
@not-my-segfault if there's anything left to do, feel free. I'm real busy today

**Pull request merged**